zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -66,7 +79,11 @@ public void configure(final Context context) {
   public void open(final Controller controller) {
     this.controller = controller;
     client = createClient();
-    // TODO createSchemaWhenNeeded();
+    searchEngineClient = new ElasticsearchEngineClient(client);
+    schemaManager = createSchemaManager();
+    schemaValidator = new IndexSchemaValidator(schemaManager);
```

Comment:
:wrench: No need to keep them as the members of the class. You can directly pass it to `schemaStartup`.